lib/c-api/build.rs

```diff
@@ -419,12 +419,15 @@ fn build_inline_c_env_vars() {
         L = shared_object_dir.clone(),
     );
 
-    // Add `-lwasmer_c_api` and, on Linux,
-    // `-rpath=<shared_object_dir>`, to the linker argument.
     println!(
-        "cargo:rustc-env=INLINE_C_RS_LDFLAGS=-lwasmer_c_api,{rpath}",
-        rpath = if cfg!(target_os = "linux") {
-            format!("-rpath={}", shared_object_dir)
+        "cargo:rustc-env=INLINE_C_RS_LDFLAGS={shared_object_dir}/libwasmer_c_api.{ext}",
```

Comment:
In the case of windows I believe is `wasmer_c_api.dll` (without the lib)

Reply:
fixed!

— https://github.com/wasmerio/wasmer/pull/1795/commits/6c9a487f2214719dee050638fd96229d8f88c09b